Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Development container for local development of the app.
+# Accompanied by a docker compose file for conveniently mounting 
+# the project directory into the container and opening ports.
+
+FROM python:3.10.12-slim
+
+RUN mkdir /workspace
+WORKDIR /workspace
+
+COPY ./requirements*.txt /workspace
+
+RUN \
+  pip install -r requirements.txt && \
+  pip install -r requirements-dev.txt
```

Comment:
Should be able to do `pip install -r requirements.txt -r requirements-dev.txt`
Though just _running_ the project should only require `requirements.txt`; the testing is what requires `requirements-dev.txt`

Reply:
> Though just running the project should only require requirements.txt; the testing is what requires requirements-dev.txt

Good point, thanks for noting.

I consider the container setup a "development setup" at the moment. As a developer, I'd like to have all dependencies available, also those for running tests.

We would need another setup for a production deployment container. But as we use toolforges' workflow for that, I figured that's not required at the moment.